app/src/main/kotlin/com/intive/picover/parties/viewmodel/PartiesViewModel.kt

```diff
@@ -38,4 +49,16 @@ class PartiesViewModel @Inject constructor(
 				}
 		}
 	}
+
+	fun updateTitle(newTitle: String) {
+		_title.value = newTitle
+	}
+
+	fun updateDescription(newDescription: String) {
+		_description.value = newDescription
+	}
```

Comment:
i don't see tests for those functions :(

Reply:
Tested added. Thank you.